### ISSUE TITLE
Does it work with Delphi 11 64Bit?

### ISSUE THREAD

**Issue by Lowlanders** [CWBudde/ComPort-Library#11]
I was wondering if the components can be compiled as 64bit?
If this is possible, how should I proceed?

**Comment by Lowlanders** [comment]
Yes, I found out, that it works on Delphi 11 64bit and also on Delphi 12 64bit.

**Comment by Vzxor** [comment]
Can you teach me how Delphi 12 works on 64-bit systems?
